pllug-presentation-system/pandoc-workflow/pandocrunner.cpp

```diff
@@ -0,0 +1,64 @@
+#include "pandocrunner.h"
+
+#include <QCoreApplication>
+#include <QDir>
+
+#include <QDebug>
+
+PandocRunner::PandocRunner(const QStringList &params, QObject* parent):
+    QProcess(parent)
+  , mParams(params)
+  , mContent("")
+  , mStatusCode(0)
+{
+    connect(this, SIGNAL(finished(int)), this, SLOT(finishedProcess(int)));
+    connect(this, SIGNAL(readyReadStandardOutput()), this, SLOT(readyReadOutput()));
+    connect(this, SIGNAL(readyReadStandardError()), this, SLOT(readyReadError()));
```

Comment:
Please, create private method "initializeConnections()" or something like that, and move all connections in this method, than call it in constructor.